### ISSUE TITLE
Alerts do not pass AA, throws contrast errors

### ISSUE THREAD

**Issue by olitharp-nci** [NCIOCPL/ncids#346]
### Issue description

The alert component is a dependency of site alert, so the alert component is imported into the documentation site. Alert throws many errors about contrast. 

> **ESTIMATE** TBD

![Screen Shot 2022-02-25 at 10 42 34 AM](https://user-images.githubusercontent.com/85901679/155753608-59d2e95e-5ec4-47e2-b5cf-5699d8f2ee31.png)
![Screen Shot 2022-02-25 at 10 42 44 AM](https://user-images.githubusercontent.com/85901679/155753632-39ede50b-340e-4a0d-9f80-a014371ee4f3.png)
![Screen Shot 2022-02-25 at 10 42 54 AM](https://user-images.githubusercontent.com/85901679/155753650-5b62f51d-220b-4964-accd-b6d20728166f.png)
![Screen Shot 2022-02-25 at 10 43 03 AM](https://user-images.githubusercontent.com/85901679/155753659-0abfdcef-bdcd-439e-8b2c-c46bd5ff2837.png)
![Screen Shot 2022-02-25 at 10 43 11 AM](https://user-images.githubusercontent.com/85901679/155753675-e10f7c77-5723-46ef-b4a7-fc915187f908.png)

```bash
Warning: [Alert (success)] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `success-lighter` background. Using `primary`. Please check your source code and project settings.
    ../node_modules/uswds/dist/scss/core/_functions.scss 1936:7                 get-color-token-from-bg()
    ../node_modules/uswds/dist/scss/core/_functions.scss 2010:27                get-link-tokens-from-bg()
    ../node_modules/uswds/dist/scss/core/mixins/_set-link-from-bg.scss 10:17    set-link-from-bg()
    ../node_modules/uswds/dist/scss/core/mixins/_alert-status-styles.scss 16:5  alert-status-styles()
    ../node_modules/uswds/dist/scss/components/_alerts.scss 93:5                @import
    src/scss/_components.scss 27:9                                              @import
    src/index.scss 22:9 
```

### Related Tickets
* #262 
* #319

**Comment by mworrest** [comment]
I have added links to the usa-alerts in sketch because USWDS excluded links from their sketch file. All links in alerts (besides the red emergency site alert) will use primary-dark as the link color. This is compliant against all background colors we've assigned for alerts.
![covid-banner-open@desktop](https://user-images.githubusercontent.com/4921685/156201292-96665a97-4f33-4469-8399-2e44717076c0.jpg)
![covid-banner-mobile](https://user-images.githubusercontent.com/4921685/156201328-1934b632-a33a-4bf2-8091-03c1c1ef4b5a.jpg)
![covid-banner](https://user-images.githubusercontent.com/4921685/156201329-b19d4eed-751b-4e1f-9e1b-66a510ccec66.jpg)

![covid-banner-open@mobile](https://user-images.githubusercontent.com/4921685/156201294-b9461b4e-9530-4d22-86d1-4416288ee028.jpg)
![covid-banner-open@tablet](https://user-images.githubusercontent.com/4921685/156201295-cb7cd9a3-9d3b-4a29-971d-26e164f0470c.jpg)
![covid-banner@tablet](https://user-images.githubusercontent.com/4921685/156201297-a97ea7f2-de55-40f8-9db1-18191eb8914d.jpg)

**Comment by bryanpizzillo** [comment]
This should be resolved by #352

**Comment by meglake** [comment]
Closing because #352 was resolved
